cluster/addons/dns/skydns-svc.yaml.in

```diff
@@ -11,6 +11,7 @@ spec:
   selector:
     k8s-app: kube-dns
   clusterIP:  {{ pillar['dns_server'] }}
+  deprecatedPublicIPs: ["10.1.4.53"]
```

Comment:
Aren't the yaml.in files processed by salt. We should be able to do the transformation via salt (so that it is conditional).
There must be an salt option for wether or not we are using contrail.